alpine-rails-nginx/Dockerfile

```diff
@@ -19,24 +21,20 @@ RUN adduser -DSH nginx \
 
 # Create system directories and service symlinks
 RUN setup-directories.sh  root  r  /etc/service/nginx /etc/service/rails \
-  && setup-directories.sh nginx r  /etc/nginx \
   && setup-directories.sh nginx rw /var/run/nginx /var/cache/nginx /var/log/nginx \
-  && setup-directories.sh rails rw /opt/rails /var/run/rails \
-  && ln -s -- /etc/sv/nginx /etc/service/nginx/run \
-  && ln -s -- /etc/sv/rails /etc/service/rails/run
+  && setup-directories.sh rails r  /opt/rails \
+  && setup-directories.sh rails rw /var/run/rails /opt/rails/log /opt/rails/tmp \
+  && ln -s /etc/sv/nginx /etc/service/nginx/run \
+  && ln -s /etc/sv/rails /etc/service/rails/run \
+  && chown rails: /usr/bin/bundle
```

Comment:
> I could change it to be executable by "other", or add an acl entry to allow it to be executed if you think that's cleaner?

I think an ACL entry is best. Because if the package we install manually comes with additional executables: They are not executable unless we whitelist these.

As these docker images are not meant for interactive use we'd even have stripped that extra executables some time. But for now its nice to whitelist whats executable.